CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to MusicBrainz Server
+
+Thank you for considering contributing to MusicBrainz Server.
+
+Get started with https://musicbrainz.org/doc/Development first,
+then continue reading with below additional guidelines.
+
+## Submitting changes
+
+### Commit message
+
+Please follow the generally accepted [seven rules of a great Git
+commit message](https://chris.beams.io/posts/git-commit/#seven-rules):
+
+1. Separate subject from body with a blank line
+1. Limit the subject line to 50 characters
+1. Capitalize the subject line
+1. Do not end the subject line with a period
+1. Use the imperative mood in the subject line
+1. Wrap the body at 72 characters
+1. Use the body to explain _what_ and _why_ vs. _how_
+
+Additionally, start the subject line with ticket reference if applicable.
+
+### Pull request
+
+#### Ticket
+
+If your change is large or relevant to users, it should have a ticket, see
+[tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20MBS),
+create one if necessary.  It will be used to follow the progress of
+the change and to generate the release notes that are made available
+to users on the blog.
+
+Untracked changes are typos, comments, coding style changes, automated
+dependency update, unnoticeable refactor, and so on.
+
+#### Title
+
+Describe your change with a short imperative title (like this)
+
+If your change relates to a ticket (see [above](#ticket)), please make sure you
+prefix your pull request title with `MBS-XXX: ` in order for our
+ticket tracker to link your pull request to that ticket, e.g.
+
+> MBS-XXX: Title
```

Comment:
Since this is meant as an example, using an actual number might be helpful to demonstrate that they shouldn't literally type XXX.
```suggestion
> MBS-1234: Title
```

Reply:
Agreed, despite the fact some people will no doubt leave MBS-1234.

Reply:
Agreed too, and it is safe to use as it is actually not in the PR template.

Reply:
Changed for each example in `CONTRIBUTING`: https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/3fe58fc29b62ae96b1115f042aa8a026033bbd54#diff-6a3371457528722a734f3c51d9238c13R48-R72